nexus/api.go

```diff
@@ -333,3 +335,32 @@ func validateLinkType(value string) error {
 	}
 	return nil
 }
+
+var durationRegexp = regexp.MustCompile("^(\\d+)(ms|s|m|h)$")
```

Comment:
I think floats should be supported. Spec says "Format of this parameter is number + unit" and I took "number" to mean float as did the Java PR.

Reply:
Yes , the server does generate floats

Reply:
Also `h` is not supported by the spec?

> Format of this header value is number + unit, where unit can be `ms` for milliseconds, `s` for seconds, and `m` for
minutes.

Reply:
Hmmm... if the server generates floats we'll need to support parsing it for compatibility but I'd stop generating floats on the server ASAP and use ints instead.
Definitely can get rid of `h`, thanks for pointing that out, I should've checked, I was looking at @cretz's comment in the Java PR (`"s", "m", or "h"`).

Reply:
Is there a reason we didn't do ISO 8601 durations?

Reply:
Just felt like overkill and we wanted a format that can be easily typed in a curl command.